certified-connectors/tyntec-WA/apiDefinition.swagger.json

```diff
@@ -75,9 +75,10 @@
 		},
 		"patch": {
 		  "responses": {},
-		  "summary": "Incoming message",
-		  "description": "Triggers on Incoming message",
-		  "operationId": "incoming",
+		  "summary": "Incoming message (deprecated)",
+		  "description": "Triggers on Incoming message (deprecated)",
+		  "operationId": "Incoming",
```

Comment:
@ridlees The operationId 'Incoming' is used multiple times. Operation identifiers must be unique.